src/app/tenant/sunbird/newLandingPage.html

```diff
@@ -0,0 +1,953 @@
+<!DOCTYPE html>
```

Comment:
@Ekta29yadav - rename the original index.html file and add these changes to the index.html